crypto/ec/ec_asn1.c

```diff
@@ -24,7 +24,7 @@ int EC_GROUP_get_basis_type(const EC_GROUP *group)
 
     /* Find the last non-zero element of group->poly[] */
     for (i = 0;
-         i < (int)OSSL_NELEM(group->poly) & group->poly[i] != 0;
+         i < (int)OSSL_NELEM(group->poly) && group->poly[i] != 0;
```

Comment:
I have also problems to add -Wundef on 1.0.2 because OSSL_NELEM is not defined there.